jquery.collection.js

```diff
@@ -357,10 +357,16 @@
 
                 var container = $(settings.container);
                 var button = collection.find('.' + settings.prefix + '-add, .' + settings.prefix + '-rescue-add, .' + settings.prefix + '-duplicate').first();
+                var secure = 0;
                 while (elements.length < settings.init_with_n_elements) {
+                    secure++;
                     var element = elements.length > 0 ? elements.last() : undefined;
                     var index = elements.length - 1;
                     elements = doAdd(container, button, collection, settings, elements, element, index, false);
+                    if(secure > settings.init_with_n_elements) {
+                        console.error('Infinite loop, element selector (' + settings.elements_selector + ') not found !!')
```

Comment:
I guess you need to add a ";" at the end of this line.
And can you remove one ! as 2 looks a bit aggressive 😄

Reply:
Yes one "!" is enough ^^

For the ";", i'll use to not use it >< 
so it's a mistake (it is not mandatory in javascript)

Reply:
oh ok, didn't know about the optional `;` 👍 thanks for the info.